### ISSUE TITLE
No longer need of ! operation for WidgetsBinding.instance

### ISSUE THREAD

**Issue by chunghha** [imaNNeoFighT/fl_chart#895]
** Don't make a duplicate issue.
You can search in issues to make sure there isn't any already opened issue with your concern.

**Describe the bug**
On flutter master channel, "flutter run" shows the warning below which is by this binding API clean-up.

https://github.com/flutter/flutter/pull/89451

**To Reproduce**

Just "fluttter run" for app using fl_chart. Might now only affect apps with master channel so at least I'd like to give a note about the warning.

**Expected behavior**
No warning below appears on console as flutter run.

```
../../../../../.pub-cache/hosted/pub.dartlang.org/fl_chart-0.41.0/lib/src/chart/pie_chart/pie_chart.dart:42:20: Warning: Operand of null-aware operation '?.' has type
'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../../../../tools/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance?.addPostFrameCallback((_) {
```

**Screenshots**
If applicable, add screenshots, or videoshots to help explain your problem.

**Versions**
 - which version of the Flutter are you using? on Master channel
 - which version of the FlChart are you using? fl_chart 0.41.0

**Comment by imaNNeoFighT** [comment]
Hi. 
It doesn't land in the `stable` channel yet.
We will fix it when it lands in the `stable` channel.

**Comment by chunghha** [comment]
Resolved as v0.50.3 released. Thanks a lot!
